universal-application-tool-0.0.1/app/services/applicant/question/RepeaterQuestion.java

```diff
@@ -45,4 +59,15 @@ public boolean isAnswered() {
         .getApplicantData()
         .hasPath(applicantQuestion.getContextualizedPath().atIndex(0).join(Scalar.ENTITY_NAME));
   }
+
+  /** Return the repeated entity names associated with this enumerator question. */
+  public ImmutableList<String> getEntityNames() {
```

Comment:
(blocking) This class is complex enough now that we should add a test class to it like [the other question classes have](https://github.com/seattle-uat/civiform/tree/main/universal-application-tool-0.0.1/test/services/applicant/question).

Reply:
adding tests!